### DISCUSSION TITLE
Getting Markor back on the play store

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#2140] [General]
I think it would be good if Markor was available on the play store. I understand that this was a huge pain for @gsantner and that he is unwilling to deal with the required time / effort / frustration.

I would like to understand the issues involved and see if there is anything I can do to enable getting Markor back on the play store. Whether this is communicating with the play store people or making requisite code changes.

As a first step I am going to try and collect all the relevant information / other discussions / tickets about this here.

I will need all the help I can get with this effort :)

**Comment by ichmoimeyo** [comment]
The latest **2.11.1** version updated on Oct 12, 2023 currently appears to be available on ...

https://github.com/gsantner/markor/releases

https://f-droid.org/en/packages/net.gsantner.markor/
AND ON
https://play.google.com/store/apps/details?id=net.gsantner.markor

even if not mentioned here on github.

(releases have gone out of sync in the past though with github and f-droid getting the latest version and the google play store lagging behind)

**Comment by harshad1** [comment]
Huh, you're right. I guess we can archive this discussion. @gsantner please let me know if / how I can help with keeping Markor on the play store.

**Reply by gsantner** [reply]
Thank you. I don't really need any help on Google Play.  
Overall most important here to me is for users and developers to respect my decision where I make Markor available.  
If it's the most favorite installation source somebody wants doesn't matter too much to me. Whatever people say, Markor is and was always public & unrestricted available for installation for anybody.
